hax/hax/handler.py

```diff
@@ -138,7 +138,8 @@ def update_process_failure(self, planner: WorkPlanner,
                     self.consul.update_process_status(
                         ConfHaProcess(
                             chp_event=event,
-                            chp_type=m0HaProcessType.M0_CONF_HA_PROCESS_M0D,
+                            chp_type=(
+                                m0HaProcessType.M0_CONF_HA_PROCESS_M0D.value),
```

Comment:
Side note:
Probably it makes sense to make `m0HaProcessType` to extend IntEnum instead of just Enum. In that case `.value` is not needed (`int(enum)` is sufficient); more info here: https://stackoverflow.com/a/49626802.

I don't think that this comment should be addressed in this PR. But probably you could add FIXME for the future?

Reply:
Addressed, thanks.